### PR TITLE
SendGrid documentation

### PR DESCRIPTION
**Pull request by mariusrusu**

### CODE REVIEW

File: source/docs/dotcom/marketplace/services/sendgrid.html.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+
+    {
+      sendgrid-n/a: [
+        {
+          name: "[service-name]",
+          label: "sendgrid-n/a",
+          plan: "free",
+          credentials: {
+            smtp_host: "smtp.sendgrid.net",
```
Comment (totof3110) [inline source/docs/dotcom/marketplace/services/sendgrid.html.md]: For consistency, "smtp_host" was changed to "hostname" (per @shalako's request).

File: source/docs/dotcom/marketplace/services/sendgrid.html.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+
+
+## Managing Services
+
+You can continue [managing your services from the command line](http://docs.cloudfoundry.com/docs/using/services/managing-services.html).
+
+
+## <a id='support'></a>Support ##
+
+All SendGrid support and runtime issues should be submitted via on of the [Contacting Service Providers for Support](http://docs.cloudfoundry.com/docs/dotcom/marketplace/contacting-service-providers-for-support.html). Any non-support related issues or product feedback is welcome at [http://support.sendgrid.com/home](http://support.sendgrid.com/home).
```
Comment (shalako) [inline source/docs/dotcom/marketplace/services/sendgrid.html.md:174]: "All SendGrid support and runtime issues should be submitted via on of the Contacting Service Providers for Support."

This sentence doesn't make sense. Should be submitted via how? 
I recommend revisiting this sentence, and moving the "Contacting Service Providers for Support" link to a new line.

File: source/docs/dotcom/marketplace/services/sendgrid.html.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        private class SMTPAuthenticator extends javax.mail.Authenticator {
+            public PasswordAuthentication getPasswordAuthentication() {
+               String username = SMTP_AUTH_USER;
+               String password = SMTP_AUTH_PWD;
+               return new PasswordAuthentication(username, password);
+            }
+        }
+    }
+```
+A sample application for using Spring Framework and SendGrid on Cloud Foundry can be found [here](https://github.com/scottfrederick/spring-sendgrid).
```
Comment (scottfrederick) [inline source/docs/dotcom/marketplace/services/sendgrid.html.md]: This has been forked into https://github.com/cloudfoundry-samples/spring-sendgrid, which is a better link to use.

File: source/docs/dotcom/marketplace/services/sendgrid.html.md [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+      :password => '<sendgrid_password>',
+      :domain => 'yourdomain.com',
+      :enable_starttls_auto => true
+    }
+```
+A sample application for using Ruby on Rails and SendGrid on Cloud Foundry can be found [here](https://github.com/laur-craciun/sendgrid-cloudfoundry-rails).
+
+## <a id='dashboard'></a>Dashboard ##
+
+SendGrid offers statistics a number of different metrics to report on what is happening with your messages.
```
Comment (scottfrederick) [inline source/docs/dotcom/marketplace/services/sendgrid.html.md:159]: "SendGrid offers statistics a number of different metrics..."

Should this just be "SendGrid offers a number of different metrics...", or something else?

File: source/docs/dotcom/marketplace/services/sendgrid.html.md (hunk 1 above)
Comment (mariusrusu) [inline source/docs/dotcom/marketplace/services/sendgrid.html.md]: done

File: source/docs/dotcom/marketplace/services/sendgrid.html.md (hunk 3 above)
Comment (mariusrusu) [inline source/docs/dotcom/marketplace/services/sendgrid.html.md]: done

**Pull request merged**